### ISSUE TITLE
Labels for all components and for connections

### ISSUE THREAD

**Issue by trstruth** [lucasepe/draft#5]
What is the reasoning behind only supporting labels on a subset of the components?  Would it be possible to extend label support to all of them?

Additionally, could connections support labels as well?

**Comment by lucasepe** [comment]
Components: Totally agree! Actually I'm working on it. In the next release each component will be able to handle labels.

Connections: they "should" already support labels (if something does not works please let me know, I'll fix it asap).

```
type Connection struct {
  Origin string `yaml:"origin"`
  Targets []struct {
    ID          string `yaml:"id"`
    Label       string `yaml:"label,omitempty"`
    Color       string `yaml:"color,omitempty"`
    Dashed      bool   `yaml:"dashed,omitempty"`
    Dir         string `yaml:"dir,omitempty"`
    Highlight   bool   `yaml:"highlight,omitempty"`
  } `yaml:"targets"`
}
```

**Comment by lucasepe** [comment]
Fixed with release [v0.7.0](https://github.com/lucasepe/draft/releases/tag/v0.7.0).
